community/kernel/src/main/java/org/neo4j/kernel/impl/index/schema/NativeSchemaIndexReader.java

```diff
@@ -47,7 +47,7 @@
     final Layout<KEY,VALUE> layout;
     private final IndexSamplingConfig samplingConfig;
 
-    protected final Set<RawCursor<Hit<KEY,VALUE>,IOException>> openSeekers;
+    final Set<RawCursor<Hit<KEY,VALUE>,IOException>> openSeekers;
```

Comment:
Let's follow _The Boy Scout Rule_ =) Could you pls rearrange fields according to their visibility?